keyboards/rgbkb/pan/pan.h

```diff
@@ -0,0 +1,37 @@
+/* Copyright 2019 coseyfannitutti
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define _x_ KC_NO
+
+#include "quantum.h"
+
+#define LAYOUT( \
+         E01,                                                          E02, \
```

Comment:
These positions are not functional because they're not assigned to a matrix position below. Are they optional?

Reply:
I was hoping to get these functional at a later date, the switches are grounded with a single pin, not on the main matrix because the switch is grounded to the RGB encoder anode

Reply:
Mostly, you could use custom matrix lite for this.  Just have the normal code, plus an additional read for the two keys. 

If you would like, I can see about whipping up some code for you